var/spack/repos/builtin/packages/gmp/package.py

```diff
@@ -28,6 +28,12 @@ class Gmp(AutotoolsPackage, GNUMirrorPackage):
     depends_on('libtool', type='build')
     depends_on('m4', type='build')
 
+    variant('static', default=False, description='Build static library')
+    variant('shared', default=True, description='Build shared library')
+
+    conflicts('~static', when='~shared', msg='Please select at least one of +static or +shared')
+    conflicts('~shared', when='~static', msg='Please select at least one of +static or +shared')
```

Comment:
I think these are equivalent and can be combined

Reply:
Done